content/building/build-versioning.md

```diff
@@ -46,3 +46,112 @@ set -x
+```
+
+{{<notebox>}}
+You can specify an optional argument `--app-store-version=APP_STORE_VERSION` to get the latest build number for a particular version of your application (`CFBundleShortVersionString`). Check the details for [get-latest-app-store-build-number](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/get-latest-app-store-build-number.md#optional-arguments-for-action-get-latest-app-store-build-number) or for [get-latest-testflight-build-number](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/get-latest-testflight-build-number.md#optional-arguments-for-action-get-latest-testflight-build-number).
+{{</notebox>}}
+
+Alternatively, if you use `YAML` configuration, you may just export the value to an environment variable and use it under your `CFBundleVersion` in `Info.plist`.
+
+## Use Codemagic CLI Tools to get the latest build number from Google Play
+
+Use [get-latest-build-number]() action from [google-play]() Codemagic CLI Tool to get the latest build number from Google Play.
+
+In order to do that, you need to provide API access to Google Play Developer API by providing `GCLOUD_SERVICE_ACCOUNT_CREDENTIALS` as arguments to the action, similarly how it's done while for [Google Play publishing](https://docs.codemagic.io/publishing-yaml/distribution/#google-play).
+
+Package name of the app in Google Play Console (Ex: com.google.example)
+
+
+Additionally, you will need to provide the package name of the app in Google Play Console (Ex. `com.google.example`).
+
+### Creating Google service account credentials
+
+You will need to set up a service account in Google Play Console and create a JSON key with credentials. See how to [set up a service account and create a key](../knowledge-base/google-play-api/#setting-up-the-service-account-on-google-play-and-google-cloud-platform).
+
+Save the API key as an [environment](../getting-started/yaml#environment) variable. Make sure to [encrypt](./encrypting/#encrypting-sensitive-data) the values of the variable before adding it to the configuration file.
+
+```yaml
+environment:
+  vars:
+    GCLOUD_SERVICE_ACCOUNT_CREDENTIALS: Encrypted(...)
+```
+
+{{<notebox>}}
+Alternatively, credentials can be specified as a command argument with the dedicated flag, see the details [here](). But anyway you should have them in environment variables so that they can be decrypted. In that case, the environment variable will be fallback for missing value in the script.
+{{</notebox>}}
+
+Once you have the Google Play Developer API access set with the mentioned above environment variable, you can get the build number using the tool
+
+```bash
+export GCLOUD_SERVICE_ACCOUNT_CREDENTIALS=$(echo $GCLOUD_SERVICE_ACCOUNT_CREDENTIALS | base64 --decode) # if you encrypted the file itself, not its content
+LATEST_BUILD_NUMBER=google-play get-latest-build-number --package-name 'com.google.example' # use your own package name
+```
```

Comment:
Like for iOS, can we define two sections for consintancy?
* `Increment build number in Flutter UI`
* `Increment build number in yaml`

We want to make it super clear how to do it in yaml and how to do it in flutter UI, i.e. in which custom build step. I'll split these up later into different files.